src/components/Map/Map.js

```diff
@@ -24,29 +24,35 @@ function Map({
   const history = useHistory();
   const [content, setContent] = useState('');
 
-  const goToStatePage = page => {
-    window.scrollTo(0, 0);
-    history.push(page);
-  };
+  const goToStatePage = React.useCallback(
+    page => {
+      window.scrollTo(0, 0);
+      history.push(page);
+    },
+    [history],
+  );
 
-  const handleClick = stateName => {
-    // externally provided click handler
-    if (onClick) {
-      return onClick(stateName);
-    }
+  const handleClick = React.useCallback(
+    stateName => {
+      // externally provided click handler
+      if (onClick) {
+        return onClick(stateName);
+      }
 
-    const stateCode = REVERSED_STATES[stateName];
+      const stateCode = REVERSED_STATES[stateName];
 
-    goToStatePage(`/us/${stateCode.toLowerCase()}`);
+      goToStatePage(`/us/${stateCode.toLowerCase()}`);
 
-    if (setMapOption) {
-      setMapOption(MAP_FILTERS.STATE);
-    }
+      if (setMapOption) {
+        setMapOption(MAP_FILTERS.STATE);
+      }
 
-    if (setMobileMenuOpen) {
-      setMobileMenuOpen(false);
-    }
-  };
+      if (setMobileMenuOpen) {
+        setMobileMenuOpen(false);
+      }
+    },
+    [onClick, goToStatePage, setMapOption, setMobileMenuOpen],
+  );
```

Comment:
I don't think this code is used anymore now that the states are links

Reply:
Oooh, interesting!  Unfortunately I tried to remove this, but got a bit nervous when I looked at the usages.  I _think_ it'd probably be fine to remove from the location page MiniMap, but our embed uses the onClick handler to redirect to the CAN homepage (rather than the state you clicked on), so I think that would break unless we rework that.

So for now I'm adding a TODO, but this really would be nice to clean up.  I just don't want to break something right now. 😬